nipype/interfaces/traits_extension.py

```diff
@@ -33,7 +33,7 @@
 from traits.api import BaseUnicode
 from traits.api import Unicode
 
-DictStrStr = traits.Dict(str, (bytes, str))
+DictStrStr = traits.Dict((bytes,str,unicode), (bytes,str,unicode))
```

Comment:
Maybe the solution is to typecast the environ somewhere in the base
interface class?

On Wed, Sep 14, 2016 at 9:00 AM, bpinsard <notifications@github.com> wrote:

> In nipype/interfaces/traits_extension.py
> <https://github.com/nipy/nipype/pull/1629#discussion_r78779207>:
>
> > @@ -33,7 +33,7 @@
> >  from traits.api import BaseUnicode
> >  from traits.api import Unicode
> >
> > -DictStrStr = traits.Dict(str, (bytes, str))
> > +DictStrStr = traits.Dict((bytes,str,unicode), (bytes,str,unicode))
>
> In the error, it seems that all my environ strings are in unicode, so
> without unicode it does not solve my problem.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/nipy/nipype/pull/1629/files/fce449863911f9512a69a35c2ae93eabb941ef8a#r78779207>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAOkp3oMdKPi2qiLTGg8QpRyp9K2DvBKks5qqBopgaJpZM4J8Jid>
> .
>